lib/widgets/recent_dm_conversations.dart

```diff
@@ -179,16 +179,39 @@ class RecentDmConversationsItem extends StatelessWidget {
     int? userIdForPresence;
     switch (narrow.otherRecipientIds) { // TODO dedupe with DM items in [InboxPage]
       case []:
-        title = TextSpan(text: store.selfUser.fullName, children: [
+        final youLabel = ZulipLocalizations.of(context).youLabel;
```

Comment:
I'd like to organize the code differently:

- Make a helper function that takes a user ID and returns a `TextSpan` with all the appropriate elements: name, then status emoji if applicable, then "(you)" if applicable.
- Use that helper function here and in the other cases.

I notice that status emojis aren't being shown in the group-DM case. This approach will cause them to be shown, which seems desirable. Since that's an independent behavior change, please make that happen in a separate commit from the commit that implements the "(you)" label.